cmd/client/cmd/app.go

```diff
@@ -157,33 +157,53 @@ var appListCmd = &cobra.Command{
 	Short:   "List all apps",
 	Long:    "Return all apps with address and team.",
 	Example: "  $ teresa app list",
-	RunE: func(cmd *cobra.Command, args []string) error {
-		tc := NewTeresa()
-		apps, err := tc.GetApps()
-		if err != nil {
-			if isNotFound(err) {
-				return newCmdError("You have no apps")
-			}
-			return err
-		}
-		// rendering app info in a table view
-		table := tablewriter.NewWriter(os.Stdout)
-		table.SetHeader([]string{"TEAM", "APP", "ADDRESS"})
-		table.SetRowLine(true)
-		table.SetAlignment(tablewriter.ALIGN_LEFT)
-		table.SetRowSeparator("-")
-		table.SetAutoWrapText(false)
-		for _, app := range apps {
-			a := ""
-			if len(app.AddressList) > 0 {
-				a = app.AddressList[0]
+	Run: appList,
+}
+
+func appList(cmd *cobra.Command, args []string) {
+	showApps, err := cmd.Flags().GetBool("show-apps")
+	if err != nil {
+		return
+	}
+
+	conn, err := connection.New(cfgFile)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Error connecting to server:", err)
+		return
+	}
+	defer conn.Close()
+
+	cli := appb.NewAppClient(conn)
+	resp, err := cli.List(context.Background(), &appb.Empty{})
+	if err != nil {
+		fmt.Fprintln(os.Stderr, client.GetErrorMsg(err))
+		return
+	}
+
+	if len(resp.Apps) == 0 {
+		fmt.Println("You do not belong to any team")
+		return
+	}
+	// rendering app list in a table view
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader([]string{"TEAM", "APP", "ADDRESS"})
+	table.SetRowLine(true)
+	table.SetAlignment(tablewriter.ALIGN_LEFT)
+	table.SetRowSeparator("-")
+	table.SetAutoWrapText(false)
+	for _, t := range resp.Apps {
+		for _, s := range []string{t.Url} {
```

Comment:
Is there a situation when one app can to have two addresses? If not, you are right and I will remove the second loop.